scss/_variables.scss

```diff
@@ -134,7 +134,7 @@ $panel-border: rgba( black, .08 ) !default;
 $popup-text: $widget-text !default;
 $popup-bg: $base-bg !default;
 $popup-border: rgba( black, .08 ) !default;
-
+$popup-box-shadow: 0 2px 4px 0 rgba(0, 0, 0, .03), 0 4px 5px 0 rgba(0, 0, 0, .04) !default;
```

Comment:
Ok, I will change the variable name. As for the color, it has beed set after discussion with @yordanov